languages/de/tl_nc_notification.xlf

```diff
@@ -130,6 +130,11 @@
         <source>Member activation</source>
         <target>Mitglied: Aktivierung</target>
         
+      </trans-unit>
+      <trans-unit id="tl_nc_notification.type.member_close.0">
+        <source>Member closing</source>
+        <target>Mitglied: Schließung</target>
+        
```

Comment:
Translations are managed on [Transifex](https://explore.transifex.com/terminal42/notification_center/) - you only need to provide the English translations and remove the other languages.